server/index.js

```diff
@@ -8,6 +8,18 @@ const ipfilter = require('express-ipfilter').IpFilter
 const errorHandler = require('./middlewares/errorHandler')
 
 let app = express()
+const Sentry = require('@sentry/node')
+const dropSensitiveEventData = require('./helpers/dropSensitiveEventData')
+
+Sentry.init({
+  dsn: 'https://43eac31915bb40caa03798a51048e756@o150853.ingest.sentry.io/5421403',
+  tracesSampleRate: 0,
+  beforeSend(event) {
+    return dropSensitiveEventData(event)
+  },
+})
+
+app.use(Sentry.Handlers.requestHandler())
```

Comment:
does this have to be up here for any reason? If not, I'd move it all the way down to line 129 where "app.use(Sentry.Handlers.errorHandler())" is written so that Sentry stuff stays together

Reply:
Thnx, great question, there was a reason for this, checkout this [link](https://docs.sentry.io/platforms/node/guides/express/). Sentry should be initialized as soon as possible, thane the rest of the app and then the error handles, at least according to this.